src/cbmc_starter_kit/template-for-repository/proofs/run-cbmc-proofs.py

```diff
@@ -233,19 +251,29 @@ def get_litani_path(proof_root):
     return proc.stdout.strip()
 
 
-def get_litani_capabilities(litani_path):
-    cmd = [litani_path, "print-capabilities"]
+def get_litani_version(litani_path):
+    cmd = [litani_path, "-V"]
```

Comment:
Not sure why this change is necessary. We'll probably revert it in the future as more capabilities are added.

Reply:
Indeed this particular change isn't necessary for these changes. I will take this commit out.